prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -66,14 +30,16 @@ def get_constraints(constraints_df, kwhmi, power, trip_strategy, location_strate
     :param int location_strategy: where the vehicle can charge-1, 2, 3, 4, 5, or 6;
         1-home only, 2-home and work related, 3-anywhere if possibile,
         4-home and school only, 5-home and work and school, 6-only work
+    :param Dict[int, Set[int]] location_allowed:
```

Comment:
Should be `:param dict location allowed:`. If necessary the type of the keys and values can be mention in the description, which is currently missing

Reply:
OK, I guess I’ve been used to using explicit types over in the ECTF repo